src/xdocs/cmdline.xml.vm

```diff
@@ -63,76 +67,65 @@ java -D&lt;property&gt;=&lt;value&gt;  \
           passed to the ClassLoader.getResource() method.
         </li>
         <li>
-          <code>-f format</code> - specify the output
-          format. Options are <code> "plain"</code> for the <a
-          href="apidocs/com/puppycrawl/tools/checkstyle/DefaultLogger.html">DefaultLogger</a>
-          and <code>"xml"</code> for the <a
-          href="apidocs/com/puppycrawl/tools/checkstyle/XMLLogger.html">XMLLogger</a>.
-          Defaults to <code>"plain"</code>.
+          <code>-f format</code> - Specifies the output
+          format. Valid values: <code>xml</code>, <code>plain</code> for <a
+          href="apidocs/com/puppycrawl/tools/checkstyle/XMLLogger.html">XMLLogger</a>
+          and <a href="apidocs/com/puppycrawl/tools/checkstyle/DefaultLogger
+          .html">DefaultLogger</a> respectively.
+          Defaults to <code>plain</code>.
         </li>
         <li>
-          <code>-p propertiesFile</code> - specify a
-          properties file to use.
+          <code>-p propertiesFile</code> - Loads the properties file.
```

Comment:
I feel a combination of how we specify options plus the original description sounds better.
 `Sets the property files to load.`

Reply:
@rnveach Done.